### PR TITLE
more debug info to the internal sync history report

### PR DESCRIPTION
**Pull request by czue**
adds columns for previous log as well as log format and errors and allows you to change the number of logs displayed

image

### CODE REVIEW

File: corehq/apps/reports/standard/deployments.py [hunk 1]
```diff
@@ -164,7 +164,10 @@ class SyncHistoryReport(DeploymentsReport):
     slug = "sync_history"
     fields = ['corehq.apps.reports.filters.users.AltPlaceholderMobileWorkerFilter']
     report_subtitles = [ugettext_noop('Shows the last (up to) 10 times a user has synced.')]
```
Comment (snopoke) [inline corehq/apps/reports/standard/deployments.py]: This subtitle is incorrect now that you can increase the limit.

**Comment by snopoke** [comment]
Nice, what happens if someone puts a huge number in the 'limit'?

**Comment by czue** [comment]
if there are lots of logs, then probably we memory spike the worker :)

(updating now)

**Comment by czue** [comment]
(fixed issues)

**Comment by snopoke** [comment]
:white_check_mark:

**Comment by czue** [comment]
@snopoke I just scoped this a bit to add error logging and viewing as well

**Comment by snopoke** [comment]
nice

**Pull request merged**